### DISCUSSION TITLE
@midwayjs/mock 引入了有安全隐患的 formidable 版本，需要升级 supertest 大版本号

### DISCUSSION THREAD

**Discussion by pilaoda** [midwayjs/midway#3808] [General / 闲聊]
https://github.com/advisories/GHSA-8cp3-66vr-3r4c

│  └─ @midwayjs/mock@npm:3.14.7 (via npm:^3.14.7)
│     └─ supertest@npm:6.3.3 (via npm:6.3.3)
│        └─ superagent@npm:8.1.2 (via npm:^8.0.5)
│           └─ formidable@npm:2.1.2 (via npm:^2.1.2)

**Comment by waitingsong** [comment]
supertest  升级到 v7 了

**Reply by czy88840616** [reply]
mock 目前只有本地开发的作用，风险还好吧。

**Reply by pilaoda** [reply]
不排除有其他线上功能会依赖formidable，导致为了适配版本不得不降级
